jax/experimental/jax2tf/tests/savedmodel_test.py

```diff
@@ -94,13 +97,8 @@ def f_jax_jvp(primals, tangents):
     self.assertAllClose(restored_model.f(x), f_jax(x))
     with tf.GradientTape() as tape:
       y = restored_model.f(xv)
-
-    # TODO: at the moment TF does not fully-support custom_gradient in a
-    # savedmodel (b/123499169), but at least a warning is printed and an
-    # exception is thrown when gradients are taken. The exception, however,
-    # is a very strange one, for now.
-    with self.assertRaisesRegex(TypeError, "An op outside of the function building code is being passed"):
-      _ = tape.gradient(y, xv)
+    self.assertAllClose(tape.gradient(y, xv).numpy(),
+                        jax.grad(f_jax)(0.7).astype(np.float32))
```

Comment:
We should use here `x` in lieu of `0.7`.

Reply:
Updated the `x` and documentation